docker-compose.yml

```diff
@@ -13,17 +13,17 @@ mysql:
     - MYSQL_DATABASE
     - MYSQL_USER
     - MYSQL_PASSWORD
-  ports:
-    - 3306:3306
+  expose:
```

Comment:
Hmm, I like the explicit nature of the override file and the flexibility to do additional customization besides just what's set forth in the core compose file. For the purposes of this PR, let's keep things as they were. However, I've opened https://github.com/ktbartholomew/saml-20-single-sign-on/issues/6 as I think embracing the override file is a good idea.